HW3/AmountOfMoneySpent.java

```diff
@@ -0,0 +1,41 @@
+package HW3;
+
+import java.util.Scanner;
+
+public class AmountOfMoneySpent {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Are you VIP client?  true/false");
+        boolean isVipClient = scanner.nextBoolean();
+
+        System.out.println("Enter your age");
+        int minage = scanner.nextInt();
+
+        int entryAmount = 5000;
+        int orderAmount;
+        int totalAmountSpent;
+
+        if (isVipClient == false && minage >= 16) {
```

Comment:
You need to check the age limitation separately at first, because if the age is under 16, then it's already unnecessary whether the client is VIP or not... BTW if a client is aged under 16, then he/she can't be a client at all, not talking about VIP client))
If checking age separately, you'll see that your code will become simpler.